src/crypto/shelley/ShelleyWallet.js

```diff
@@ -595,10 +572,8 @@ export default class ShelleyWallet extends Wallet implements WalletInterface {
       withdrawals: [
         {
           addressing: this._getRewardAddressAddressing(),
-          rewardAddress: Buffer.from(
-            await (await this.getRewardAddress()).to_bytes(),
-            'hex',
-          ).toString('hex'),
+          // $FlowFixMe flow thinks rewardAddressHex can be null
```

Comment:
You can usually fix this by doing something like
```javascript
const { rewardAddressHex} = this;
if (rewardAddressHex== null) throw new Error('reward address is null')
```

the fact it's non-null should propagate correctly in that case.